addons/mail/static/src/messaging/component/notification_list/notification_list_tests.js

```diff
@@ -0,0 +1,272 @@
+odoo.define('mail.messaging.component.NotificationListTests', function (require) {
+'use strict';
+
+const components = {
+    NotificationList: require('mail.messaging.component.NotificationList'),
+};
+
+const {
+    afterEach: utilsAfterEach,
+    afterNextRender,
+    beforeEach: utilsBeforeEach,
+    pause,
+    start: utilsStart,
+} = require('mail.messaging.testUtils');
+
+QUnit.module('mail', {}, function () {
+QUnit.module('messaging', {}, function () {
+QUnit.module('component', {}, function () {
+QUnit.module('NotificationList', {
+    beforeEach() {
+        utilsBeforeEach(this);
+
+        this.data['mail.channel'] = {
+                fields: {
+                    channel_type: {
+                        string: "Channel Type",
+                        type: "selection",
+                    },
+                    id: {
+                        string: "Id",
+                        type: 'integer',
+                    },
+                    message_unread_counter: {
+                        string: "# unread messages",
+                        type: 'integer',
+                    },
+                    name: {
+                        string: "Name",
+                        type: "char",
+                        required: true,
+                    },
+                },
+            };
+        this.data['mail.message'] = {
```

Comment:
all these data field definitions should be in `messaging/testUtils:beforeEach()` helper function